chatbot.js

```diff
@@ -0,0 +1,30 @@
+
+// I created a variable to hold my index and initialized it as an object.
+var myObject = {
```

Comment:
You can use `var` if you like but there are some issues with using `var` so generally, it's preferred to use `const` or `let`. [Here's](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e) an article that explains some of the issues with using var.